.github/workflows/linter.yml

```diff
@@ -43,3 +43,21 @@ jobs:
 
     - name: Run Lint
       run: ./scripts/lint.sh --go-lint
+
+  format-solidity:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
+    steps:
+      - name: Checkout repository and submodules
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Install Foundry
+        run: ./scripts/install_foundry.sh
+      
+      - name: Check Solidity Formatting
+        if: success() || failure()
```

Comment:
What's the purpose of this conditional? If it's pertaining to the previous lint jobs, I'd expect it to be directly under `format-solidity`, not in the step.

Reply:
Whoops, didn't need this either.